src/components/Editor/Editor.css

```diff
@@ -135,6 +135,14 @@ html[dir="rtl"] .editor-mount {
   100% { background-color: transparent; }
 }
 
+.highlight-line {
+  background-color: green;
+}
+
+.theme-dark .highlight-line {
+  background-color: tomato;
```

Comment:
I haven't decided on colors yet, so ignore this. Was thinking about the debug colors but with opacity, maybe?